index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
Well, that makes sense. For now I leave it like that because of the ongoing discussion.